File: MY PROGRAMS/LB/CPP/SELECTION,SEQUENCE AND  ITERATION/a10q2.cpp
/* Write a program which accept width & height of rectangle from user and calculate its area. (Area = Width * Height)

Input:	Width:5.3 
	Hieght:9.78
Output:	51.834

*/




#include<iostream>

using namespace std;

class RectangleArea
{
	public:
		double Return(double,double);
};

double RectangleArea::Return(double dwt,double dht)
{
	double dar=0.0;

	dar=dwt*dht;

	return(dar);
}

int main()
{
	double dvalue1=0.0,dvalue2=0.0,dret=0.0;

	cout<<"Enter the width and hieght of the rectangle:";
	cin>>dvalue1>>dvalue2;

	RectangleArea raobj;

	dret=raobj.Return(dvalue1,dvalue2);

	cout<<"Area of a rectangle is "<<dret;

	return(0);
}

File: MY PROGRAMS/LB/JAVA/NUMBER/a10q2.java
/*	2. Write a program which accept width & height of rectangle from user and calculate its area. (Area = Width * Height).

	Input:	5.3	9.78
	Output:	51.834	*/




import java.util.*;

class Assignment10
{
	public double Return(double dwt,double dht)
	{
		double dar=0.0;

		dar=dwt*dht;

		return(dar);
	}
}

class a10q2
{
	public static void main(String arg[])
	{
		Scanner sobj=new Scanner(System.in);

		double dvalue1=0.0,dvalue2=0.0,dret=0.0;

		System.out.println("Enter the width and height of the rectangle: ");
		dvalue1=sobj.nextDouble();
		dvalue2=sobj.nextDouble();

		Assignment10 aobj=new Assignment10();

		dret=aobj.Return(dvalue1,dvalue2);

		System.out.println("Area of the rectangle is "+dret);
	}
}
